support/http/main.go

```diff
@@ -17,20 +17,24 @@ import (
 	"gopkg.in/tylerb/graceful.v1"
 )
 
-// DefaultListenAddr represents the default address and port on which a server
+// defaultListenAddr represents the default address and port on which a server
 // will listen, provided it is not overridden by setting the `ListenAddr` field
 // on a `Config` struct.
-const DefaultListenAddr = "0.0.0.0:8080"
-
-// DefaultShutdownGracePeriod represents the default time in which the running
-// process will allow outstanding http requests to complete before aborting
-// them.  It will be used when a grace period of 0 is used, which normally
-// signifies "no timeout" to our graceful shutdown package.  We choose not to
-// provide a "no timeout" mode at present.  Feel free to set the value to a year
-// or something if you need a timeout that is effectively "no timeout"; We
-// believe that most servers should use a sane timeout and prefer one for the
-// default configuration.
-const DefaultShutdownGracePeriod = 10 * time.Second
+const defaultListenAddr = "0.0.0.0:8080"
+
+const (
+	// defaultShutdownGracePeriod represents the default time in which the running
+	// process will allow outstanding http requests to complete before aborting
+	// them.  It will be used when a grace period of 0 is used, which normally
+	// signifies "no timeout" to our graceful shutdown package.  We choose not to
+	// provide a "no timeout" mode at present.  Feel free to set the value to a year
+	// or something if you need a timeout that is effectively "no timeout"; We
+	// believe that most servers should use a sane timeout and prefer one for the
+	// default configuration.
+	defaultShutdownGracePeriod = 10 * time.Second
+
+	defaultReadTimeout = 5 * time.Second
+)
```

Comment:
These constants are relatively unrelated and I wouldn't group these. We don't group the listen address above with them.

```suggestion
// defaultShutdownGracePeriod represents the default time in which the running
// process will allow outstanding http requests to complete before aborting
// them.  It will be used when a grace period of 0 is used, which normally
// signifies "no timeout" to our graceful shutdown package.  We choose not to
// provide a "no timeout" mode at present.  Feel free to set the value to a year
// or something if you need a timeout that is effectively "no timeout"; We
// believe that most servers should use a sane timeout and prefer one for the
// default configuration.
const defaultShutdownGracePeriod = 10 * time.Second

const defaultReadTimeout = 5 * time.Second
```

Reply:
https://github.com/stellar/go/pull/3061/commits/5a468136124052ddc0034ca0ec4e0412c1f41474